docker/server/start.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Why this start script? Can't we just use RUN and CMD here in the Dockerfile?

Reply:
As the startup script contains multiple commands that need to be run *in* the container when starting up (eg creating the configuration file based on an environment variable) we can not use RUN.
CMD could be done, but imo multiple piped commands make it messy while a startup script is easy to read/change.

Reply:
Ack, got it.

Reply:
Ack